gf-webapp/src/logic/rooms/roomPage/RoomPage.tsx

```diff
@@ -30,19 +29,24 @@ export const RoomPage = () => {
     setRoom(currentRoom);
   };
 
+  const [isRoomInitialized, setIsRoomInitialized] = useState(true);
+
   useEffect(() => {
-    if (id) {
-      loadRoom(id);
-    }
+    (async () => {
+      if (id) {
+        await loadRoom(id);
+        setIsRoomInitialized(false);
+      }
+    })();
   }, [id]);
 
   // if data not initialized yet
   if (!room) {
-    return <Loader />;
+    return <PageBorder isLoading={isRoomInitialized} />;
```

Comment:
I needed to render the PageBorder here, because until the rooms are loaded no further code is executed.